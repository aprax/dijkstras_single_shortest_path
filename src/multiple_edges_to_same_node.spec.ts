import findShortestPath from '.';
import type { Nodes } from './types';

const nodes: Nodes = new Map([
	['A', [['B', 2], ['B', 7], ['B', 5]]],
])
it('should find the shortest among the multiple edges that point to the same node', () => {
	expect(findShortestPath('A', nodes)).toMatchSnapshot();
})