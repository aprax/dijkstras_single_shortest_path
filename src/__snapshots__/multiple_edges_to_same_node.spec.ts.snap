// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should find the shortest among the multiple edges that point to the same node 1`] = `
Map {
  "A" => Array [
    "A",
    0,
  ],
  "B" => Array [
    "A",
    2,
  ],
}
`;
